nessie/lib/util.py

```diff
@@ -159,7 +159,8 @@ def resolve_sql_template_string(template_string, **kwargs):
         'rds_schema_asc': app.config['RDS_SCHEMA_ASC'],
         'rds_schema_boac': app.config['RDS_SCHEMA_BOAC'],
         'rds_schema_coe': app.config['RDS_SCHEMA_COE'],
-        'rds_schema_e_i': app.config['RDS_SCHEMA_E_I'],
+        'rds_schema_': app.config['RDS_SCHEMA_DATA_SCIENCE'],
```

Comment:
@lyttam Did something go awry here? Looks like we lost E_I and got a dupe DATA_SCIENCE.